src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -35,6 +37,12 @@ var (
 				return fmt.Errorf("k8s cluster already bootstrapped")
 			}
 
+			const minTimeout = 3 * time.Second
```

Comment:
Sometimes `k8s bootstrap` times out with 30s on low machines. Added the `--timeout` flag and by that, increased the timeout to 90s.
(I still would advocate for a generic timeout flag for all commands ...)